dftimewolf/lib/preflights/cloud_token.py

```diff
@@ -37,4 +40,38 @@ def CleanUp(self):
     """Carries out optional cleanup actions at the end of the recipe run."""
 
 
-modules_manager.ModulesManager.RegisterModule(GCPTokenCheck)
+class AWSAccountCheck(module.PreflightModule):
+  """Checks for AWS authentication."""
+
+  def SetUp(self, profile_name=None):  # pylint: disable=arguments-differ
+    """Runs a boto3 check to ensure AWS authentication is configured
+
+    Args:
+      profile_name (str): Optional. The profile to test.
+    """
+
+    try:
+      session = boto_session.Session(profile_name=profile_name)
+      client = session.client('sts')
```

Comment:
What does `sts` stand for here? Could you clarify that in a comment?

Reply:
Added more detail to the method docstring.